packages/react-components/react-migration-v0-v9/etc/react-migration-v0-v9.api.md

```diff
@@ -9,6 +9,14 @@ import { ObjectOf } from '@fluentui/react-northstar';
 import { ObjectShorthandValue } from '@fluentui/react-northstar';
 import * as React_2 from 'react';
 
+// @public (undocumented)
+export const buttonMigrationStyles: {
+    v9Icon: () => GriffelStyle;
+    v9CustomSizeIcon: () => GriffelStyle;
+    v0Icon: () => GriffelStyle;
```

Comment:
v0 returns a different type, it should be `ICSSInJSStyle` from `@fluentui/react-northstar`